processors.go

```diff
@@ -211,6 +212,12 @@ func ProcessorRandomDate(cmap *ColumnMapper, input string) (string, error) {
 	return scrambledDate, nil
 }
 
+// ProcessorRandomDigits will return a random string of digit(s) that is >= 0.4 Jaro-Winkler similar than the input but
```

Comment:
We are not using Jaro-Winkler any more since it can make reconstruction of the data easier through ML. You can remove that from this comment.

Reply:
Removed 👍